sites/fast-component-explorer/app/explorer.tsx

```diff
@@ -361,12 +346,67 @@ class Explorer extends Foundation<
         }
     }
 
-    private handlePopState = (): void => {
+    private handleCanvasOverlayTrigger = (): void => {
+        this.setState({
+            mobileFormVisible: false,
+            mobileNavigationVisible: false,
+        });
+    };
+
+    private handleWindowPopState = (): void => {
         if (window.location.pathname !== this.state.locationPathname) {
             this.handleUpdateRoute(window.location.pathname);
         }
     };
 
+    private handleWindowResize = (): void => {
+        if (this.editorContainerRef.current) {
+            if (this.windowResizing) {
+                clearTimeout(this.windowResizing);
+            }
+
+            this.windowResizing = setTimeout(() => {
+                this.setState({
+                    width: 0,
+                    height: 0,
+                });
+
+                this.setViewerToFullSize();
+
+                if (this.state.activePivotTab === "code") {
+                    this.createMonacoEditor();
+                }
+            }, 800);
```

Comment:
Is 800 a magic number? Can you add some docs about *why* 800?

Reply:
Just tested this, I had added it because I was having an issue which turned out to be state related but I forgot to remove this.